Engine/RealTime/BacktestingRealTimeHandler.cs

```diff
@@ -173,10 +171,7 @@ public void AddEvent(RealTimeEvent newEvent)
         /// </summary>
         public void ScanEvents()
         {
-            for (var i = 0; i < _events.Count; i++)
-            {
-                _events[i].Scan(_time);
-            }
+            _events.ForEach(x => x.Scan(_time));
```

Comment:
The for loop is technically faster since it doesn't have to perform an extra method invocation in the loop